pyrato/parametric.py

```diff
@@ -151,3 +151,37 @@ def air_attenuation_coefficient(
             ) * 20.0 / np.log(10.0) / (np.log10(np.exp(1.0)) * 10.0))
 
     return air_abs_coeff
+
+def critical_distance(
+                     volume,
+                     reverberation_time):
+    r"""Calculate the critical distance of a room with
+    given volume and reverberation time [#kra]_.
+    Assumes the source directivity is 1 (omnidirectional source).
```

Comment:
I would be good practice, but not manorytory, to have a short summary in the first line and than more details to in a new paragraph like e.g. this:
```python
"""
This is a short summary.

Give more details and even references here _[#].

```